source/openapi-admin-v3.yaml

```diff
@@ -1989,6 +1989,41 @@ paths:
       responses:
         "404":
           description: No IP addresses or CIDR blocks to delete
+  "/groups/{groupId}/apps/{appId}/metrics":
+    get:
+      tags:
+        - metrics
+      operationId: adminGetMetrics
+      summary: Retrieve App Services metrics
+      parameters:
+        - "$ref": "#/components/parameters/GroupId"
+        - "$ref": "#/components/parameters/AppId"
+        - name: start
+          in: query
+          description: |-
+            The ISO 8601 date and time of the start of the query period.
+          schema:
+            type: string
+          required: true
+        - name: end
+          in: query
+          description: |-
+            The ISO 8601 date and time of the end of the query period.
+          schema:
+            type: string
+          required: true
+        - name: granularity
+          in: query
+          description: |-
+            Specifies the granularity of the query period as an ISO-8601
+            duration value. Examples include "P31D" (31 day), "PT1H" (1 hour),
+            and "PT5M" (5 minutes).
```

Comment:
Do we have a list of these granularity strings somewhere? Maybe a link out to other docs or (even better IMO) an `enum` in the schema that lists all the options.

Reply:
It's a format actually specified in ISO-8601, so we can't really enumerate all of the options. I gave a few examples here but we could link out to Wikipedia?